src/include/OSL/llvm_util.h

```diff
@@ -354,36 +354,30 @@ class OSLEXECPUBLIC LLVM_Util {
     /// Generate code for a call to the function pointer, with the given
     /// arg list.  Return an llvm::Value* corresponding to the return
     /// value of the function, if any.
-    llvm::Value *call_function (llvm::Value *func,
-                                llvm::Value **args, int nargs);
+    llvm::Value *call_function (llvm::Value *func, OIIO::cspan<llvm::Value *> args);
```

Comment:
Let's change these to `cspan` (since we're in the OSL namespace already -- oslconfig.h already creates the right `using` type alias).

Reply:
Sure, didn't notice the alias.